roles/config-httpd/tasks/prep.yml

```diff
@@ -3,7 +3,7 @@
 - name: 'Install required packages'
   package:
     name: '{{ item }}'
-    state: installed
+    state: latest
```

Comment:
This is the opposite of what we discussed :-) - i.e.: if anything uses `latest` it should be changed to `installed`. This is to avoid "surprised updates" of packages during a config/content update run. Please change back to `installed`